eventonica-api/models/event.js

```diff
@@ -0,0 +1,31 @@
+const { Sequelize, DataTypes, Model } = require('sequelize');
```

Comment:
One thing I was looking for, what is the command for initializing your db? 

It would be nice if you added this command to a script in `package.json`. Even nicer if you included a readme for the `eventonica-api` project with instructions about how to interact with the DB.

I think the start script would be a good idea for you to _actually_ do. I understand if adding a readme isn't super helpful/important for you right now considering you're the only one who really works on this project!